spec/support/mockers/mocker.rb

```diff
@@ -0,0 +1,69 @@
+require 'rspec/mocks/standalone'
+
+class Mocker < RSpec::Mocks::InstanceVerifyingDouble
```

Comment:
This class is pretty awesome, but also fairly complex and I think it could use a little more explicit documentation, particularly geared towards novice Rails devs. That said, it's a really nifty extension of `InstanceVerifyingDouble` and I can see how we'd use it to build other mockers.

Out of curiosity, were there any existing gems that you looked at to solve this problem and found lacking for some reason?